linter_driver_rustc/Cargo.toml

```diff
@@ -11,3 +11,6 @@ linter_api = { path = "../linter_api", features = ["driver-api"] }
 linter_adapter = { path = "../linter_adapter" }
 
 bumpalo = "3.9.1"
+
+[package.metadata.rust-analyzer]
+rustc_private=true
```

Comment:
Did this fix it, that's funny; I had the same problem where rust-analyzer quit giving me IntelliSense for rustc and tried the same fix but couldn't get it working.

Reply:
Yeah, rs can be a bit iffy with rustc, this fixed it for me :upside_down_face: